modules/cms-kit/src/Volo.CmsKit.Admin.Application.Contracts/Volo/CmsKit/Admin/Permissions/AdminPermissions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Volo.Abp.Reflection;
+
+namespace Volo.CmsKit.Admin.Permissions
+{
+    public class AdminPermissions
+    {
+        public const string GroupName = "Admin";
```

Comment:
I think this is not a good name. `Admin` and `Public` is too common.has no special meaning.

`CmsKit.Admin` is better.

Reply:
If it is a CMS framework or component set like hikalkan's introduction, it may provide abstract entity class, derived repository base class, derived AppService class, UI components, and more.

If so, maybe `AdminPermissions`, `AdminMenu`, `RemoteServiceName` and their classes should not be defined, since it is not an application module and is used to create some other modules or apps.

Reply:
Right, 

updated by https://github.com/abpframework/abp/commit/6d402e47186bf70028dc60854d902cf90de4a31e

Reply:
This is only preparing initial solution. This will get a lot of update soon. :)

Reply:
@cotur 
`AdminMenu`, `RemoteServiceName` should also be changed.